`app-page-layout-tests` failing with timeout

## Description

Beginning yesterday with `securedrop_ci` run 2984, `app-page-layout-tests` is failing with a timeout in `make validate-test-html`.

## Steps to Reproduce

- https://app.circleci.com/pipelines/github/freedomofpress/securedrop?branch=5828-reply-message
- https://app.circleci.com/pipelines/github/freedomofpress/securedrop?branch=pull%2F6086

## Expected Behavior

`app-page-layout-tests` always passes after `make validate-test-html || true`.

## Actual Behavior

<https://app.circleci.com/pipelines/github/freedomofpress/securedrop/2987/workflows/b2712c98-1396-41a8-9781-430471382e4e/jobs/56712/parallel-runs/1?filterBy=FAILED>:

```
███ Validating HTML source from 72 page-layout test(s)
make: *** [Makefile:231: validate-test-html] Hangup

Too long with no output (exceeded 10m0s): context deadline exceeded
```

<https://app.circleci.com/pipelines/github/freedomofpress/securedrop/2987/workflows/25a47bb5-8bf5-4510-bad1-3edef0e01271/jobs/56716/parallel-runs/1?filterBy=ALL>:

```███ Validating HTML source from 32 page-layout test(s)
make: *** wait: No child processes.  Stop.
make: *** Waiting for unfinished jobs....
make: *** wait: No child processes.  Stop.

Too long with no output (exceeded 10m0s): context deadline exceeded
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -151,7 +151,10 @@ jobs:
       # "informational" status---in other words, fail if validation fails!
       - run:
           name: Validate HTML (informational)
-          command: make validate-test-html || true
+          command: |
+            fromtag=$(docker images |grep securedrop-test-focal-py3 |head -n1 |awk '{print $2}')
+            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-focal-py3:${fromtag:-latest}" \
+              make validate-test-html || true
 
       - store_test_results:
           path: ~/project/test-results

